### PR TITLE
Fixing broken preconditions

### PR DESCRIPTION
**Pull request by mrprescott**

### CODE REVIEW

**Comment by cstamas** [comment]
+1

**Comment by mcculls** [comment]
+1

**Comment by kellyrob99** [comment]
+0 My preference is to avoid PRs for things like this - I more than trust the team to push trivial cleanup like this without the need for review, JUST DO IT! ;)

File: components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java [hunk 1]
```diff
@@ -256,8 +255,7 @@ protected void setMetadata(final T entity, final ODocument doc) {
    * Get metadata from entity.
    */
   protected EntityMetadata getMetadata(final T entity) {
-    checkNotNull(entity != null);
-    assert entity != null;
+    checkNotNull(entity);
```
Comment (jdillon) [inline components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java]: btw these looked like they might have been mean to have been checkState?  cnn works though

**Comment by mrprescott** [comment]
http://bamboo.s/browse/NX3-OSSF184-1 :white_check_mark:

File: components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java (hunk 1 above)
Comment (jdillon) [inline components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java]: ftr the assert here was probably to keep IDEA from complaining... I've yet to figure out how to make it learn what Preconditions do :-(

Comment (cstamas) [inline components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java]: Latest update of IDEA learned Preconditions as I see ;)

**Comment by adreghiciu** [comment]
+1

**Pull request merged**